.env.template

```diff
@@ -154,7 +154,7 @@ OPENAI_API_KEY=your-openai-api-key
 
 ### STABLE DIFFUSION WEBUI
 ## SD_WEBUI_AUTH - Stable diffusion webui username:password pair (Example: username:password)
-## SD_WEBUI_URL - Stable diffusion webui API URL (Example: http://127.0.0.1:7860)
+## SD_WEBUI_URL - Stable diffusion webui API URL (Example: http://127.0.0.1:7860 or http://host.docker.internal:7860 in case you run autogpt in a docker container)
```

Comment:
If I remember this isn't the same across platforms for windows, macos, and linux. Can you confirm?